src/components/ColorPicker/ColorPickerItem/ColorPickerItem.st.css

```diff
@@ -57,5 +57,21 @@
     height: calc(100% - 2px);
     border-radius: value(colorPickerBorderRadius);
     border: 1px solid #C9C9C9;
+    position: relative;
+    overflow: hidden;
 }
 
+.root:isCrossedOut .radioInner::before {
+    content: '';
+    position: absolute;
+    border-top: 1px solid white;
```

Comment:
Note that the `border-top` color here, should be the same as the as the border there ☝️  (#c9c9c9)